examples/aws_ebs/README.md

```diff
@@ -1,9 +1,16 @@
 This is a simple web server pod which serves HTML from an AWS EBS
 volume.
 
-Create a volume in the same region as your node add your volume
-information in the pod description file aws-ebs-web.yaml then create
-the pod:
+First make sure that your minions have the following IAM permissions : (http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html#create-iam-role-console):
```

Comment:
Could you make a descriptive name for the page you've linked to and then make this just a markdown link?

Example: `[text describing linked page](url#section)`